R/doAzureParallel.R

```diff
@@ -177,11 +177,23 @@ getparentenv <- function(pkgname) {
       uploadBlob(id, system.file(startupFolderName, "worker.R", package="rAzureBatch"))
       uploadBlob(id, system.file(startupFolderName, "merger.R", package="rAzureBatch"))
 
+      resourceFiles <- list()
+      if(!is.null(obj$options$azure$resourceFiles)){
+        resourceFiles <- obj$options$azure$resourceFiles
+      }
+
+      if(!is.null(obj$options$azure$resourcefiles)){
+        resourceFiles <- obj$options$azure$resourcefiles
+      }
+
       sasToken <- constructSas("2016-11-30", "r", "c", id, storageCredentials$key)
-      resourceFiles <- list(generateResourceFile(storageCredentials$name, id, "splitter.R", sasToken),
+      staticResourceFiles <- list(generateResourceFile(storageCredentials$name, id, "splitter.R", sasToken),
```

Comment:
Should we call it jobResourceFiles? It's not really static since the job gets deleted. The resource files will also be deleted?